Samples/Client/SwiftClient/SwiftClientUITests/SwiftClientUITests.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  SwiftClientUITests.swift
+//  SwiftClientUITests
+//
+//  Created by Michael Perel on 9/24/17.
+//  Copyright © 2017 Michael Perel. All rights reserved.
+//
+
+import XCTest
+
+class SwiftClientUITests: XCTestCase {
```

Comment:
are you planning to add these tests? @michaelperel

Reply:
Cocoa Pods automatically generates these. I plan on writing these but need a little bit of guidance as to what we should test specifically. The plan was to make sure the requests from the iOS client are the same as the requests from the web client, assuming that the web client is the canonical reference.

Reply:
from the ui perspective, i think we'd like to see an e2e that can confirm video is coming through, but i think that's out of scope for now. I'll defer to your judgment and knowledge of the ui components here, but if there's an opportunity for us to validate their behavior fairly quickly, i'd love to include those tests here (we can talk more about what these might look like, if that helps)